src/widgets/requirements/ui/requirements.test.tsx

```diff
@@ -35,6 +37,6 @@ describe('Requirements', () => {
     const button = screen.getByRole('link', { name: /Register as a mentor/i });
 
     expect(button).toBeVisible();
```

Comment:
I think this is redundant because it's check in `LinkCustom` component

Reply:
Done